packages/shared/components/pills/Pill.svelte

```diff
@@ -8,14 +8,14 @@
     export let uppercase: boolean = false
     export let invisible: boolean = false
     export let backgroundColor: string | undefined = undefined
-    export let rounded: string = 'rounded-2xl'
+    export let borderRadius: PillBorderRadius = PillBorderRadius['2xl']
 </script>
 
 <pill-wrapper
     class:shrink-0={shrink}
     class:uppercase
     class:invisible
-    class="flex items-center {backgroundColor ?? ''} {rounded} px-2.5 py-1"
+    class="flex items-center {backgroundColor ?? ''} rounded-{borderRadius} px-2.5 py-1"
```

Comment:
I don't know if this is prioritary or not but string concatenation should be [avoided for tailwind classes](https://iotaledger.github.io/firefly/guides/coding-conventions/components/#:~:text=When%20writing%20tailwind%20classes%2C%20string%20concatenation%20should%20be%20avoided%20because%20the%20tailwind%20purging%20process%20skips%20string%20concatenations)